deploy/stage-internal-test-job.yaml

```diff
@@ -44,6 +44,10 @@ objects:
             value: ${IQE_TEST_IMPORTANCE}
           - name: IQE_PARALLEL_WORKER_COUNT
             value: ${IQE_PARALLEL_WORKER_COUNT}
+          - name: IQE_RP_ARGS
+            value: "--reportportal"
```

Comment:
This will be added automatically, see https://gitlab.cee.redhat.com/insights-qe/iqe-core/-/blob/master/iqe/scripting/container_commands/run.py#L182 we need to pass args for --report-portal param

Reply:
ah okay, I was referring https://gitlab.cee.redhat.com/insights-qe/iqe-core/-/blob/master/iqe/scripting/container_commands/run.py?ref_type=heads#L412 which seems wrong